crates/subspace-farmer/src/single_disk_farm.rs

```diff
@@ -0,0 +1,30 @@
+use crate::single_plot_farm::SinglePlotPieceGetter;
+use subspace_core_primitives::{Piece, PieceIndex, PieceIndexHash};
+
+/// Abstraction that can get pieces out of internal plots
+#[derive(Debug, Clone)]
+pub struct SingleDiskFarmPieceGetter {
+    single_plot_piece_getters: Vec<SinglePlotPieceGetter>,
+}
+
+impl SingleDiskFarmPieceGetter {
+    /// Create new piece getter for many single plot farms
+    pub fn new(single_plot_piece_getter: Vec<SinglePlotPieceGetter>) -> Self {
+        Self {
+            single_plot_piece_getters: single_plot_piece_getter,
```

Comment:
Minor thing. It is better to rename `single_plot_piece_getter` to `single_plot_piece_getters` :)

Reply:
Yes, must have left after some refactoring